docs/team/lightz96.adoc

```diff
@@ -1,7 +1,120 @@
+** What it does: It indicates the type of filter used currently for the transaction list.
+** Justification: If users have forgotten about the filter they set for the transaction list, they can refer to this filter text.
+
+* *Code contributed*: Please click this link to view a sample of my code: link:https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=lightz96&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false[[RepoSense Code Contribution Dashboard]].
+
+* *Other contributions*
+** Project management
+*** Created the GUI prototype for THRIFT. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/59[#59])
+*** Refactored the entire code base by renaming the Person to Transaction, AddressBook to THRIFT. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/76[#76])
+*** Managed issues assigned to me from version 1.0 to version 1.4 (https://github.com/AY1920S1-CS2103T-W12-2/main/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+assignee%3Alightz96+[Issues assigned])
+** Enhancement to existing features
+*** Revamped the entire user interface. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/193[#193])
+*** Added the ability to view the usage of a specific command. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/187[#187])
+** Documentation
+*** Inserted content from external files (E.g. SettingUp.adoc) into Developer Guide and changed to THRIFT context. (Pull Request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/233[#233])
+** Community
+*** Reviewed Pull Requests (with non-trivial review comments): https://github.com/AY1920S1-CS2103T-W12-2/main/pull/272[#272], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/211[#211], https://github.com/AY1920S1-CS2103T-W12-2/main/pull/143[#143]
+** Tools
+*** Integrated https://ci.appveyor.com/project/lightz96/main[AppVeyor] (Continuous Integration tool), https://app.netlify.com/sites/thrift/deploys[Netlify] (Deployment Preview tool), https://coveralls.io/github/AY1920S1-CS2103T-W12-2/main[Coveralls] (Code Coverage tool) to the project. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/46[#46])
+
+[[UserTag]]
+== Contribution to User Guide
+
+The following section demonstrates my documentation skills in guiding users to execute the command.
+
+(Start of extract)
+
+include::../UserGuide.adoc[tag=undo]
+
+(End of extract)
+
+Please click on these links to view other sections that I have also contributed to the User Guide: https://ay1920s1-cs2103t-w12-2.github.io/main/UserGuide.html#RedoTag[Redoing the undone command], https://ay1920s1-cs2103t-w12-2.github.io/main/UserGuide.html#HelpTag[Displaying help page]
+
+[[DeveloperTag]]
+== Contribution to Developer Guide
+
+The following section demonstrates my documentation skills in providing technical details regarding the implementation of the undo and redo features.
```

Comment:
You might want:
... technical details _such as_ the implementation ...

Reply:
Fixed